run_all_checks.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -eux
+
+cargo fmt --all -- --check
+cargo test --workspace
+cargo clippy --workspace --all-targets
+./build_examples.sh
```

Comment:
This file is missing an EOL character on its last line.

Reply:
Fixed. I also change invalid `rust` to `cargo` in the travis config.